.github/dependabot.yml

```diff
@@ -1,16 +1,15 @@
 ---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
 version: 2
 updates:
-  - package-ecosystem: pip
-    directory: /
-    schedule:
-      interval: daily
-    labels:
-      - skip-changelog
-    versioning-strategy: lockfile-only
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: daily
-    labels:
-      - "skip-changelog"
+      interval: "weekly"
```

Comment:
Can't we just change the interval instead of reformatting and reshuffling everything?

Reply:
Hmm, that wasn't my intention, my assumption was somehow that there was no dependabot config so far.

Reply:
@felixfontein I think that changing daily to weekly or even quarterly would be benefitial, though.

Reply:
Do you worry about execution time for dependabot? Or about the frequency of PRs? Since there have been a total of five PRs since November 2022, I don't think the frequency of PRs is a problem.

Reply:
The frequency. I'd like the PRs to happen rarely.

Reply:
Hmm, how about to address this when it turns out there are too many? So far we had an average of <0.6 PRs per month.

Reply:
Quarterly would be better. I'm not comparing one repo frequency, but in general, I get many notifications from other repos, which is a different metric, motivating me to cut on the dep update noise.